site/content/en/docs/Installation/_index.md

```diff
@@ -264,7 +264,7 @@ We can install Agones to the cluster using the
 
 ```bash
 kubectl create namespace agones-system
-kubectl apply -f https://github.com/GoogleCloudPlatform/agones/raw/release-0.7.0/install/yaml/install.yaml
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/master/install/yaml/install.yaml
```

Comment:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/{{< release-branch >}}/install/yaml/install.yaml
```

Nice pickup!

Looks like we missed a spot when doing the dynamic update (we actually don't want to point at master, as master is development,  and development could always be broken)

Reply:
Not sure if you meant to include the suggested change in the PR? But we need the change to accept it.